raiden/network/proxies/token_network.py

```diff
@@ -856,8 +895,13 @@ def withdraw(
             raise ValueError(msg)
 
         with self.channel_operations_lock[partner]:
+            # gaslimit below must be defined
+            raise NotImplementedError('feature temporarily disabled')
```

Comment:
I'm torn about raising an unrecoverable error here: 
If this is to guard developers from calling this code path, I would rather raise it at the head of `withdraw(...)`.
If it is really only about the missing gas_limit definition, then I would only issue a warning and not let the client crash.
Generally speaking, I assume this is dead code in the current client stage (unsupported by the contracts). If this is the case, I suggest to create or amend to the issue of the feature, delete the code and leave it to the git gods to recover it when necessary.

Reply:
I'll remove withdraw related code in a different PR